### PR TITLE
create off-canvas documentation

### PR DESCRIPTION
**Pull request by Mady-Camara**

### CODE REVIEW

File: Blazor_Foundation_6_Test_Shared/Documentation/Containers/Off-Canvas.razor [hunk 1]
```diff
@@ -1,6 +1,416 @@
...
+        <p>
+            The Off-canvas component requires a positioning attribute to determine which side of the viewport it opens from:
+            <ul>
+                <li><Code>position-left</Code></li>
+                <li><Code>position-right</Code></li>
+                <li><Code>position-top</Code></li>
+                <li><Code>position-bottom</Code></li>
+            </ul>
+        </p>
+        <HighlightCS Language="html" Full="true" HeaderIcon="mdi-language-html5" HeaderBGcolor="#5d2b90" HeaderTitle="Razor"
```
Comment (maxgsamson) [inline Blazor_Foundation_6_Test_Shared/Documentation/Containers/Off-Canvas.razor:161]: The code example has an img tag artifact either add an image or code that part of the code.

File: Blazor_Foundation_6_Test_Shared/Documentation/Containers/Off-Canvas.razor [hunk 2]
```diff
@@ -1,6 +1,416 @@
...
+                     Content="PE9mZkNhbnZhc1dyYXBwZXI+CiAgPE9mZkNhbnZhcyBQb3NpdGlvbj0icG9zaXRpb24tbGVmdCIgSWQ9Im9mZkNhbnZhcyI+CiAgICA8IS0tIFlvdXIgbWVudSBvciBPZmYtY2FudmFzIGNvbnRlbnQgZ29lcyBoZXJlIC0tPgogIDwvT2ZmQ2FudmFzPgogIDxPZmZDYW52YXNDb250ZW50PgogICAgPCEtLSBZb3VyIHBhZ2UgY29udGVudCBsaXZlcyBoZXJlIC0tPgogIDwvT2ZmQ2FudmFzQ29udGVudD4KPC9PZmZDYW52YXNXcmFwcGVyPg==" />
+
+        <h2>Click Triggers</h2>
+        <p>
+            To create a click trigger that opens the panel, add the attribute <Code>DataToggle</Code> to component <Code>Button</Code> or <Code>data-open</Code> or  to any element. That element will then open or toggle the panel when clicked on. The value of the data attribute should be the ID of the off-canvas.
+        </p>
+        <HighlightCS Language="html" Full="true" HeaderIcon="mdi-language-html5" HeaderBGcolor="#5d2b90" HeaderTitle="Razor"
+                     Content="PEJ1dHRvbiBEYXRhVG9nZ2xlPSJvZmZDYW52YXMiPk9wZW4gTWVudTwvQnV0dG9uPg==" />
+        <Button DataToggle="offCanvas1">Open Menu</Button>
```
Comment (maxgsamson) [inline Blazor_Foundation_6_Test_Shared/Documentation/Containers/Off-Canvas.razor:64]: Remove that button we dont need to see a button if doesn't do anything in that context.

**Review by maxgsamson — CHANGES_REQUESTED** [review]
@Mady-Camara still some changes same as last time @A-San96 .

File: Blazor_Foundation_6_Test_Shared/Documentation/Containers/Off-Canvas.razor (hunk 2 above)
Comment (Mady-Camara) [reply Blazor_Foundation_6_Test_Shared/Documentation/Containers/Off-Canvas.razor:64]:
```suggestion
```

**Pull request closed without merging**